src/components/sdk/ContentShare/ContentShare.mdx

```diff
@@ -19,26 +23,85 @@ import { ContentShare } from 'amazon-chime-sdk-component-library-react';
 
 ## Usage
 
+### With Single Content Share
+
 ```jsx
 import React from 'react';
 import {
   MeetingProvider,
   ContentShare,
-  useContentShareControls
+  useContentShareControls,
 } from 'amazon-chime-sdk-component-library-react';
 
 const App = () => {
   const { toggleContentShare } = useContentShareControls();
 
   return (
     <MeetingProvider>
-      <ContentShare nameplate='Content share' />
+      <ContentShare nameplate="Content share" />
```

Comment:
Are we using " or ' in this repo?

Reply:
We use single quote for JS, but here it should be double quote because it's a property on JSX element. Like we should use double quote on HTML element